wallet-next/src/storage.rs

```diff
@@ -39,6 +39,14 @@ impl Storage {
         Ok(result.map(|row| row.height as u64))
     }
 
+    pub async fn set_last_sync_height(&self, height: i64) -> anyhow::Result<()> {
```

Comment:
Should this be incorporated into `record_block`, so any block recorded with height > latest will update the `sync_height' table?

Reply:
Yeah, I think so, my instinct is that we probably don't want to have any methods that set individual parts of the storage, because we know that at the moment, we only need to (1) initialize it or (2) ratchet forward a block at a time